src/networking.c

```diff
@@ -2510,6 +2503,80 @@ int processEventsWhileBlocked(void) {
     return count;
 }
 
+/* ==========================================================================
+ * Threaded Conn
+ * ========================================================================== */
+
+pthread_t conn_thread;
+pthread_mutex_t conn_thread_mutex;
+
+typedef struct {
+    int fd;
+    int flags;
+} Jobs;
+
+void pushConnToCreateClient(int fd, int flags) {
+    pthread_mutex_lock(&conn_thread_mutex);
+    Jobs* job = zmalloc(sizeof(Jobs));
+    job->fd = fd;
+    job->flags = flags;
+    listAddNodeTail(server.clients_pending_create, job);
+    pthread_mutex_unlock(&conn_thread_mutex);
```

Comment:
Maybe we should to weak up the main thread to handle this client connection. Just like https://github.com/antirez/redis/blob/unstable/src/module.c#L3657-L3660